src/js/actions/transform.js

```diff
@@ -209,12 +212,13 @@ define(function (require, exports) {
      *
      * @param {Document} document
      * @param {Layer} targetLayer
-     * @param {{w: number, h: number}} size
+     * @param {{w=: number, h=: number}} size
```

Comment:
I think I will separate them for this case to further iterate that both do not have to be set, and in a lot of cases they in fact do not occur simultaneously. I know the "=" means the same but further clarification in transform.js doesn't hurt anyone in my opinion.